backend/src/packages/chaiNNer_standard/image/video_frames/save_video.py

```diff
@@ -323,16 +353,50 @@ def save_video_node(
     _: None,
     save_dir: Path,
     video_name: str,
+    simplicity: Simplicity,
     container: VideoFormat,
     encoder: VideoEncoder,
     video_preset: VideoPreset,
     crf: int,
     advanced: bool,
     additional_parameters: str | None,
```

Comment:
You still use `additional_parameters` in simple mode. See the "Append additional parameters" section below.